CheckSammy.py

```diff
@@ -438,86 +444,85 @@ class CheckSammy():
     def join_path(self,a,b):
         return a + '/' + b
 
-    def calc_md5(self, file):
+    def calculate_md5(self, file):
         # Calculates the md5 for the selected file.
         h = hashlib.md5()
 
-        with open(file, 'rb') as tohash:
-            for chunk in iter(lambda: tohash.read(4096), b''):
+        with open(file, 'rb') as file_data:
+            for chunk in iter(lambda: file_data.read(4096), b''):
                 h.update(chunk)
 
         return(h.hexdigest())
 
-    def check_md5(self, ff, switch=0):
-        # Calculates a new checksum and confronts it with the one stored in the json.
+    def check_md5(self, path, switch=0):
```

Comment:
The arg previously called ``ff` seemed to be a full path to me - let me know if I messed this up.